data/crac-io/crac-io-json/src/main/java/com/farao_community/farao/data/crac_io_json/JsonExport.java

```diff
@@ -41,6 +54,14 @@ public void exportCrac(Crac crac, OutputStream outputStream) {
             ObjectMapper objectMapper = createObjectMapper();
             objectMapper.registerModule(new Jdk8Module());
             objectMapper.setSerializationInclusion(JsonInclude.Include.NON_NULL);
+            SimpleModule module = new SimpleModule();
+            module.addSerializer(FreeToUse.class, new FreeToUseSerializer());
+            module.addSerializer(OnConstraint.class, new OnConstraintSerializer());
+            module.addSerializer(OnContingency.class, new OnContingencySerializer());
+            module.addSerializer(PstWithRange.class, new PstWithRangeSerializer());
+            module.addSerializer(AlignedRangeAction.class, new AlignedRangeActionSerializer());
+            module.addSerializer(ComplexNetworkAction.class, new ComplexNetworkActionSerializer());
```

Comment:
We could put all this in crac-impl, in a CracImplJsonModule, which extends SimpleModule.
And here only do `objectMapper.registerModule(new CracImpJsonModule());`

Reply:
:ok: